src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private double length;
+
+    public Square(String color, double length) {
+        super(color);
+        this.length = length;
+    }
+
+    @Override
+    public double getArea() {
+        return length * length;
+    }
+
+    @Override
+    public void draw() {
+        System.out.printf("Figure: square, area: %.2f sq. cm, length: %.2f cm, color: %s.%n",
```

Comment:
Avoid using System.out.printf for drawing the figure. It's better to override the `toString` method for the representation of the object and then print the object using `System.out.println`. This will separate the representation logic from the output operation.